Sadot/User/UserHomePage.cs

```diff
@@ -49,16 +88,48 @@ private void FillTableList()
             tables = db.GetTablesData();
             for (int i = 0; i < tables.Length; i++)
             {
-                dgvTableList.Rows.Add(tables[i].TableID, tables[i].TableStatus);
+                if (tables[i].OrderState == "הזמנה בהכנה")
+                {
+                    DateTime waitingTime = DateTime.Now - tables[i].TimeOfOrder.TimeOfDay;
+                    dgvTableList.Rows.Add(tables[i].TableID, tables[i].TableStatus, tables[i].OrderState, waitingTime.ToString("mm:ss"));
+                }
+                else
+                    dgvTableList.Rows.Add(tables[i].TableID, tables[i].TableStatus, tables[i].OrderState, tables[i].TimeOfOrder.ToString("mm:ss"));
```

Comment:
Break Line

Reply:
Fixed
I have rewritten the function - review it again, please